azure-pipelines.yml

```diff
@@ -76,6 +76,9 @@ jobs:
       Python38:
         python.version: '3.8'
         python.org.version: '3.8.2'
+      Python39:
+        python.version: '3.9'
+        python.org.version: '3.9.0'
```

Comment:
Any reason not to do the linux wheel as well?

Reply:
I have to update docker image. Dont have the rights though on DockerHub

Reply:
Ahh. I had forgotten about Docker:)